js/liveaw.js

```diff
@@ -76,11 +76,12 @@ function list_matches(s, cb) {
 }
 
 function on_edit_event(s) {
-	var match_players = {};
-	s.event.matches.forEach(function(match) {
+	var match_players = s.event.matches.reduce(function(prev, match) {
```

Comment:
With the initial object, I meant the `{}` being passed into reduce. After all, this change *adds* complexity instead of removing it - the line count will be the same, but one has to understand that how reduce is called here.

 But after looking at the context in detail, `reduce` may indeed be more appropriate.